### PR TITLE
f (refs ADO-16632): make docxImporter available in addons.

### PR DESCRIPTION
**Pull request by MoritzMandler**
Ticket:
https://www.dev.diplanung.de/DefaultCollection/BOP-HH/_workitems/edit/16632

Description:
Make the docxImporter docx-To-HTML available in addons.

### How to review/test
run: ```composer u demos-europe/demosplan-addon```

### Linked PRs (optional)
https://github.com/demos-europe/demosplan-addon/pull/115

- [ ] Tests updated/created
- [ ] Update documentation
- [x] Link all relevant tickets
- [ ] Move the tickets on the board accordingly
- [ ] Data-Cy attributes added/updated ([conventions](https://dplan-documentation.demos-europe.eu/development/guidelines-conventions/coding-styleguides/twig_html.html#guideline-for-naming-cypress-hooks))

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
         "cocur/slugify": "^4.0",
         "composer/package-versions-deprecated": "1.11.99.5",
         "demos-europe/edt-extra": "^0.24.0",
-        "demos-europe/demosplan-addon": "^0.24",
+        "demos-europe/demosplan-addon": "dev-f_ADO_15514_docxAddonImporterInterface",
```
Comment (mussbach) [inline composer.json:36]: needs to be adjusted in final version, ok for now

File: tests/backend/core/Import/DocxImportToolTest.php [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+        $this->sut = self::getContainer()->get(DocxAddonImporterInterface::class);
+    }
+
+    public function testDocxImport()
+    {
+        $filePath = __DIR__ . '/MoritzTollesAnschreiben.docx';
```
Comment (mussbach) [inline tests/backend/core/Import/DocxImportToolTest.php:23]: no user specific data please

**Review by mussbach — CHANGES_REQUESTED** [review]
some early minor remarks

**Comment by MoritzMandler** [comment]
not needed for the chosen plan.

**Pull request closed without merging**